WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt

```diff
@@ -37,26 +37,64 @@ k_sparta = {
 	capital = 478 # Sparta
 
 	culture = tsakonian
-	religion = hellenic_pagan
 
 	creation_requires_capital = yes
 
 	allow = {
-		AND = {
-			independent = yes
-			culture = tsakonian
-			is_adult = yes
-			prestige = 10000
-			realm_size = 10
+		always = no # Requires Special Decision
+	}
+
+	d_three_hundred = {
+		color = { 55 40 50 }
+		color2 = { 99 39 49 }
+
+		capital = 478 # Sparta
+
+		allow = {
 			OR = {
-				religion = hellenic_pagan
-				religion = cybelian_pagan
-				religion = neoplatonist
-			}
-			NOT = {
-				tier = emperor
-			}
+				culture = spartan
+				culture = tsakonian
+			}
+			custom_tooltip = {
+				name = is_greek_and_hellenic
+				hidden_tooltip = {
+					AND = {
+						culture = greek
+						OR = {
+							religion = hellenic_pagan
+							religion = cybelian_pagan
+							religion = neoplatonist
+							religion = mithraic_pagan
+							religion = mastimani
+						}
+					}
+				}
+			}
+
+			religion_group = pagan_group
+
+			hidden_tooltip = { always = no # Requires special Decision }
```

Comment:
No, if I did that, then it would be Tsakonian OR Greek and Hellenic OR in the Pagan Group, allowing all Pagans to reform Sparta!